### DISCUSSION TITLE
Changing the name to only bird bc hummingbird is long

### DISCUSSION THREAD

**Discussion by HosMercury** [hummingbird-project/hummingbird#193] [Ideas]
Also The HB which is everywhere is not elegant. Could be Bird .

**Comment by tib** [comment]
https://www.youtube.com/watch?v=8GEKjBEm0SA 😅

**Comment by adam-fowler** [comment]
Hummingbird is not renaming to Bird (even if it is a word). 

Here is a previous thread about the HB prefix https://github.com/hummingbird-project/hummingbird/issues/186
